sygnal/gcmpushkin.py

```diff
@@ -700,13 +706,17 @@ def _build_data(
                 # Truncate fields to a sensible maximum length. If the whole
                 # body is too long, GCM will reject it.
                 if data[attr] is not None and len(data[attr]) > MAX_BYTES_PER_FIELD:
+                    overflow_fields += 1
                     data[attr] = data[attr][0:MAX_BYTES_PER_FIELD]
 
         if api_version is APIVersion.V1:
             if isinstance(data.get("content"), dict):
                 for attr, value in data["content"].items():
                     if not isinstance(value, str):
                         continue
+                    if len(value) > MAX_BYTES_PER_FIELD:
```

Comment:
Are these bytes or strings? Looking at it python will count code points for strings.

Reply:
Ah, they are indeed code points. Good catch.